src/app/Caches/Create/Features/FeatureAlert.tsx

```diff
@@ -1,26 +1,26 @@
-import {Alert, AlertActionLink, AlertVariant} from "@patternfly/react-core";
+import { Alert, AlertActionLink, AlertVariant } from "@patternfly/react-core";
 import React from "react";
-import {CacheFeature} from "@services/infinispanRefData";
-import {useTranslation} from "react-i18next";
-import {useCreateCache} from "@app/services/createCacheHook";
+import { CacheFeature } from "@services/infinispanRefData";
+import { useTranslation } from "react-i18next";
+import { useCreateCache } from "@app/services/createCacheHook";
 
-const FeatureAlert = (props: {feature: CacheFeature}) => {
+const FeatureAlert = (props: { feature: CacheFeature, errorType?: string, helperText?: string }) => {
```

Comment:
I prefer that this alter gets the cache feature and the error to be directly displayed instead of these properties.
So if the error to be displayed is given as a parameter we simply display the error, otherwise we get use only the feature to display the proper error.